cypress/integration/accessibility_Video_spec.js

```diff
@@ -8,7 +8,7 @@ describe('Video Accessibility check', () => {
     cy.configureAxe({
       rules: [
         {
-          id: 'color-contrast',
+          id: 'aria-progressbar-name',
```

Comment:
New check by axe-core. Not fixing it as part of this diff but let's make sure we don't introduce this issue in other components.